src/handlers.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-return-assign */
```

Comment:
- [ ] There are cases where disabling a linter rules is almost inevitable. But I think this is not one of those. Instead, let's rewrite lines where it complains about something. Lastly, remember that you can always google how to fix linter rules.

Tip for this case, we don't want to return assignments like `return x = y`, instead we want to assign first and the return:
```
x = y;
return x;
```

Reply:
Thank you so much for your keen observation. I have managed to solve the concerns in my code that you requested and everything works fine.